docs/src/pages/components/data-grid/state/state.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Data Grid - State
+---
+
+# Data Grid - State
+
+<p class="description">Initialize and read the state of the data grid.</p>
+
+## Initialize the state
+
+Some state keys can be initialized with the `initialState` prop.
+
+> ⚠️ The `initialState` can only be used to set the initial value of the state, the grid will not react if you change the `initialState` value later on.
+>
+> If you need to fully control some models, use the control props instead (e.g. `prop.filterModel` or `prop.sortModel`).
+> More information on each feature documentation page.
+
+{{"demo": "pages/components/data-grid/state/InitialState.js", "bg": "inline", "hideToolbar": true}}
```

Comment:
```suggestion
{{"demo": "pages/components/data-grid/state/InitialState.js", "bg": "inline"}}
```
Otherwise, we cannot see how you initialise the state